tests/cases/datetime/lt_datetime.test

```diff
@@ -0,0 +1,25 @@
+### SUBSTRAIT_SCALAR_TEST: 1.0
+### SUBSTRAIT_INCLUDE: /extensions/functions_datetime.yaml
+
+# timestamps: examples using the timestamp type
+lt(2016-12-31 13:30:15::timestamp, 2017-12-31 13:30:15::timestamp) = True::boolean
+lt(2018-12-31 13:30:15::timestamp, 2017-12-31 13:30:15::timestamp) = False::boolean
+
+# timestamp_tz: examples using the timestamp_tz type
+lt(1999-01-08 01:05:05-08:00::timestamp_tz, 1999-01-08 04:05:06-05:00::timestamp_tz) = True::boolean
+lt(1999-01-08 01:05:06-08:00::timestamp_tz, 1999-01-08 04:05:06-05:00::timestamp_tz) = False::boolean
```

Comment:
Should we use `ts_tz` instead of `timestamp_tz` in the tests?

Reply:
changed it